client/EditContributionView.js

```diff
@@ -342,12 +342,19 @@ class EditContributionView extends Component {
                             label={approvalText}
                             labelPlacement="bottom"
                             control={
+                                ((this.props.publishedList && contrib && (this.props.publishedList[contrib.ref.id] === 'true'))) ?
+                                <Switch
```

Comment:
`disabled={(this.props.publishedList && contrib && (this.props.publishedList[contrib.ref.id] === 'true')}`

No need for ternary, can do inline on disabled prop